epoch/api/README.md

```diff
@@ -73,6 +78,40 @@ An event has the same type as a response, except for not having a tag:
 | action | string | what was the action | Yes |
 | payload | object | data from the action | Yes |
 
+## Channels WebSocket API definition
+
+### Description
+Channels provide means for off chain transactions with functionality of on
+chain dispute resolution.
+Channels require persisted connections to Aeternity nodes. Each participant in
+a channels uses one's own trusted node. For persistence of this connection websockets
+are used.
+Channels have on chain state that persists who the participants are and the
+total amout of tokens put in the channel.
+Each channel also has an off chain state representing the latest distribution of the balance of the
+channel. It can be updates - each new state is co-signed by both parties and only then it becomes the latest valid state of the
+channel. At any point of time channel can be closed either unilaterally or
+with a mutual agreement.
+
+### Connection
+The epoch node supports an endpoint with a configurable port where the
+websocket's clients connect. It is located on `/channel`.
+
+The node could serve multiple channel websocket clients. Their number is configured in
+the `epoch.yaml`. When all websocket connections are consumed - any new incoming
+connections will be queued. The queue has a maximum size and when it is
+reached - any new incoming connections will be rejected with an error code 400.
+This is to prevent the node of being overloaded with websocket connections.
+
+### General message types
+All messages have the format:
+
+| Name | Type | Description | Required |
+| ---- | ---- | ----------- | -------- |
+| action | string | what is the action | Yes |
+| tag | string | additional info for the action | No |
+| payload | object | data for action | Yes |
+
 ### List of WS APIs
```

Comment:
This change makes the pseudo schema specific to oracle and chain part of the "Channels WebSocket API definition" section - that makes no sense. Please move this newly introduced "Channels WebSocket API definition" section below the `  * [Chain WS API](./chain_ws_api.md)` line.

Reply:
Please see https://github.com/aeternity/protocol/pull/76